python/hsfs/core/feature_group_engine.py

```diff
@@ -38,9 +38,23 @@ def save(self, feature_group, feature_dataframe, write_options):
                 feat.primary = True
             if feat.name in feature_group.partition_key:
                 feat.partition = True
+            if (
+                feature_group.hudi_precombine_key is not None
+                and feat.name in feature_group.hudi_precombine_key
```

Comment:
Why using the `in` here. precombine_key can be at most one string, just compare 2 strings.